cmd/login/runner_test.go

```diff
@@ -153,11 +171,31 @@ func TestMCLoginWithInstallation(t *testing.T) {
 			flags: &flag{
 				KeepContext: true,
 			},
+			token: "token",
+		},
+		// OIDC flow
+		// This case is skipped in the CI because the oidc function enforces opening a browser window
+		// TODO: refactor the function to not enforce that anymore, run this test case in CI
+		{
+			name: "case 4",
```

Comment:
As described above, we skip this case in the CI for now. Refactoring the OIDC function so that browser window opening is not enforced is the next step